cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/ExportXmlCompare.java

```diff
@@ -605,7 +605,12 @@ static ComparisonResult ignoringRdfChildLookupEquivalentInjection(Comparison com
     }
 
     static ComparisonResult ignoringRdfChildLookupStaticVarCompensator(Comparison comparison, ComparisonResult result) {
-        if (result == ComparisonResult.DIFFERENT && comparison.getType() == ComparisonType.CHILD_LOOKUP) {
+        if (result == ComparisonResult.DIFFERENT && comparison.getType() == ComparisonType.TEXT_VALUE) {
```

Comment:
To make the intent explicit, maybe split in two separate methods named `ignoringStaticVarCompensatorControlEnabled` and `ignoringStaticVarCompensatorQ`, and check for text values. Ignore difference in text values if:
```
cxpath.contains("StaticVarCompensator") && cxpath.contains(".controlEnabled")
```
and for the second method ignore difference in text values if:
```
cxpath.contains("StaticVarCompensator.q")
```

Reply:
Done